css/interviewprep.css

```diff
@@ -0,0 +1,447 @@
+body {
+	margin:0px auto;
+	padding:0;
+	background: #222;
+	font-family: "Source Sans Pro","sans-serif";
+	font-size: 1.0em;
+	color:#666;
+  position: relative;
+}
+* {
+	margin:0;
+	padding:0;
+}
+
+
+/** element defaults **/
+/*table {
+	width:100%;
+	text-align:left;
+}
+th, td {
+	padding:5px 10px;
+}
+th {
+	color:#fff;
+	border-top:1px solid #222;
+	background-color:#2F2F2F;
+}
+td {
+	border-bottom:1px solid #f4f4f4;
+}*/
+code, blockquote {
+	display:block;
+	border-left:5px solid #ddd;
+	padding:10px;
+	margin-bottom:20px;
+}
+blockquote p {
+	font-style:italic;
+	font-family:Georgia, "Times New Roman", Times, serif;
+	margin:0;
+	height: 1%;
+}
+p {
+	line-height:2.0em;
+	margin-bottom:20px;
+}
+a {
+	color:#00BFFF;
+}
+a:hover {
+	color:#000;
+}
+a:focus {
+	outline:none;
+}
+a.button {
+    background-color: #00BFFF;
+    border-radius: 5px;
+    color: #FFFFFF;
+    display: inline-block;
+    font-weight: bold;
+    padding: 8px 22px;
+    font-size: 0.8em;
+    letter-spacing: 0.25px;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+a.button-reversed {
+    background-color: #333;
+}
+
+a.button:hover {
+	background-color: #0000CD;
+}
+
+a.button-reversed:hover {
+	background-color: #222;
+}
+fieldset {
+	display:block;
+	border:none;
+	border-top:1px solid #e0e0e0;
+}
+fieldset legend {
+	font-weight:bold;
+	padding-right:10px;
+	color:#999;
+}
+fieldset form {
+	padding-top:15px;
+}
+fieldset p label {
+	float:left;
+	width:150px;
+	font-family: "Source Sans Pro","sans-serif";
+}
+fieldset form input, fieldset form select, fieldset form textarea {
+	padding:5px;
+	color:#333333;
+	font-size:13px;
+	font-family: "Source Sans Pro","sans-serif";
+	border:1px solid #ddd;
+}
+fieldset form input.formbutton {
+border:none;
+    background-color: #80B763;
+    border-radius: 5px;
+    color: #FFFFFF;
+    display: inline-block;
+    font-weight: bold;
+    padding: 8px 22px;
+    font-size: 0.8em;
+    letter-spacing: 0.25px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+
+}
+h1 {
+	font-size:2.4em;
+}
+h2 {
+	color:#2f2f2f;
+	font-size:1.8em;
+	font-weight:bold;
+	padding:0 0 5px;
+	margin:0;
+}
+h3 {
+	color:#0000CD;
+	font-size:1.6em;
+	padding-bottom:10px;
+}
+h4 {
+	padding-bottom:10px;
+	font-size:1.4em;
+	color:#000;
+}
+h5 {
+	padding-bottom:10px;
+	font-size:1.2em;
+	color:#666666;
+}
+ul, ol {
+	margin:0 0 35px 35px;
+}
+li {
+	padding-bottom:5px;
+}
+
+
+/** wrapper **/
+div#wrapper {
+	margin:0px auto;
+	padding:0;
+}
+
+.width {
+	width:auto;
+	margin: 0 auto;
+}
+
+
+
+
+/** header **/
+header {
+	padding:30px 17px 30px;
+	margin:0 auto;
+}
+header h1 {
+	text-align: right;
+}
+
+header h1 a,
+header h1 a:hover,
+header h1 a:visited {
+	color: #fff;
+	font-size: 1.5em;
+	text-decoration: none;
+	letter-spacing:-2px;
+}
+
+header h2 {
+	color:#fff;
+	text-align: right;
+	font-size: 0.8em;
+	font-weight: normal;
+	padding-bottom:0;
+	letter-spacing:0;
+}
+
+
+/** body **/
+section#body {
```

Comment:
Any way to change the body to be a static background pic, like that of the circuit board we have on the main Catalyst site? Or add a gradient? If not, can we change this to smth less jarring by adding in a bit more red/green, like #2060cd?